### PR TITLE
[misc] Add new issues templates

### PR DESCRIPTION
**Pull request by sjwsl**
Related issue = #3385

### CODE REVIEW

**Review by strongoier — COMMENTED** [review]
Should `question.md` be placed under `.github/ISSUE_TEMPLATE/`?

**Comment by sjwsl** [comment]
> Should `question.md` be placed under `.github/ISSUE_TEMPLATE/`?

Ah sorry, this should be a draft PR. Not finished yet.

**Comment by strongoier** [comment]
> > Should `question.md` be placed under `.github/ISSUE_TEMPLATE/`?
> 
> Ah sorry, this should be a draft PR. Not finished yet.

You can click "convert to draft" :-)

**Comment by sjwsl** [comment]
> > > Should `question.md` be placed under `.github/ISSUE_TEMPLATE/`?
> > 
> > 
> > Ah sorry, this should be a draft PR. Not finished yet.
> 
> You can click "convert to draft" :-)

Now it's done! Thanks anyway :)

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+name: Ask a Question
+about: Ask anything about Taichi
+title: ''
+labels: question
+assignees: ''
+
+---
+
+<!--
+Before asking a question, These are what you should have tried:
```
Comment (strongoier) [inline .github/ISSUE_TEMPLATE/question.md:11]:
```suggestion
Before asking a question, these are what you should have tried:
```

Comment (qiao-bo) [inline .github/ISSUE_TEMPLATE/question.md:11]:
```suggestion
Before asking a question, please first consider:
```

File: .github/ISSUE_TEMPLATE/document.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+name: Doc
+about: Have doubts or suggestions about Taichi Docs
+title: ''
+labels: doc
+assignees: ''
+
+---
+
+<!--
+Please describe any errors, ambiguities, and any areas that you think can be improved in the documentation.
```
Comment (strongoier) [inline .github/ISSUE_TEMPLATE/document.md:11]:
```suggestion
Please describe any errors, ambiguities, or other improvement opportunities that you can find in the documentation.
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+name: Ask a Question
+about: Ask anything about Taichi
+title: ''
+labels: question
+assignees: ''
+
+---
+
+<!--
+Before asking a question, Please first consider:
```
Comment (strongoier) [inline .github/ISSUE_TEMPLATE/question.md:11]:
```suggestion
Before asking a question, please first consider:
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+---
+
+<!--
+Before asking a question, please first consider:
+
+- Searched Google
+- Searched [existing issues](https://github.com/taichi-dev/taichi/issues)
+- Searched [Taichi Doc](https://docs.taichi.graphics/)
+- Searched [Taichi forum](https://forum.taichi.graphics/)
```
Comment (strongoier) [inline .github/ISSUE_TEMPLATE/question.md:16]:
```suggestion
- Searching Google
- Searching [existing issues](https://github.com/taichi-dev/taichi/issues)
- Searching [Taichi Doc](https://docs.taichi.graphics/)
- Searching [Taichi Forum](https://forum.taichi.graphics/)
```

**Review by strongoier — APPROVED** [review]
LGTM!

**Pull request merged**